test/addons-napi/test_error/test.js

```diff
@@ -55,3 +55,61 @@ assert.strictEqual(test_error.checkError({}), false,
 // Test that non-error primitive is correctly classed
 assert.strictEqual(test_error.checkError('non-object'), false,
                    'Non-error primitive correctly classed by napi_is_error');
+
+try {
+  test_error.throwExistingError();
```

Comment:
If `throwExistingError()` and similar calls below don't actually throw an error, then this test would incorrectly pass, right?